### PR TITLE
[Doc enhancement][Add info][Issue3923]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d9b0294b5015f552c3ee094d06a81fbfaa46e662/msteams-platform/concepts/build-and-test/deep-links.md) | 83 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cc1ca2bb-19ec-49d1-b7e1-001f138b0ab2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f6b5a5ee3e1b51e5c8f82b72c7737f8d953ea2bd/msteams-platform/concepts/build-and-test/deep-links.md) | 83 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50a296a2-ea83-4fe7-9bf3-e0fbb97be25d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc6797e38e29b09c0fffe4238037c857a3932eab/msteams-platform/concepts/build-and-test/deep-links.md) | 83 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6eeb2b80-8c02-4a87-9829-378495be2e23) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 1]
```diff
@@ -151,13 +151,19 @@ The query parameters are:
...
+* `groupId`: Group ID of the file, ae063b79-5315-4ddb-ba70-27328ba6c31e 
 
-Following is the sample format of deeplink to files:
+> [!NOTE]
+> You can see `threadId` and `groupId` in the URL from the channel.  
+
+The following deep link format is used in a bot, connector, or messaging extension card:
+`https://teams.microsoft.com/l/file/5E0154FC-F2B4-4DA5-8CDA-F096E72C0A80?tenantId=<tenantid>&fileType=<filetype>&objectURL=<objectURL>&baseUrl=<baseURL>&serviceName=<Name>&threadId=<threadid>&groupId=<groupId>`
+
+Following is the example format of deeplink to files:
```
Comment (v-abirade) [inline msteams-platform/concepts/build-and-test/deep-links.md:166]: The following example...

**Review by v-abirade — CHANGES_REQUESTED** [review]
As discussed.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/687b69dd0a544a48106364d1ce92a29d1cc67745/msteams-platform/concepts/build-and-test/deep-links.md) | 83 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fadc5d77-a7ab-435a-8421-8db629db20cd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-links.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ The query parameters are:
 The following deep link format is used in a bot, connector, or messaging extension card:
 `https://teams.microsoft.com/l/file/5E0154FC-F2B4-4DA5-8CDA-F096E72C0A80?tenantId=<tenantid>&fileType=<filetype>&objectURL=<objectURL>&baseUrl=<baseURL>&serviceName=<Name>&threadId=<threadid>&groupId=<groupId>`
 
-Following is the example format of deeplink to files:
+The following is the example format of deeplink to files:
```
Comment (v-abirade) [inline msteams-platform/concepts/build-and-test/deep-links.md:166]: The following example format shows the deeplink to files:

**Review by v-abirade — CHANGES_REQUESTED** [review]
@Rajeshwari-v,

Please update the sentence.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-links.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1bf378d77c6ca10dd80129621d0a5e72dbd49dcd/msteams-platform/concepts/build-and-test/deep-links.md) | 83 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f3fd8db3-5b8c-47c9-83e8-15a150c3329e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**